matchzoo/preprocessor/cdssm_preprocessor.py

```diff
@@ -130,7 +151,10 @@ def transform(
                 text = unit.transform(text)
             text = [ngram_unit.transform([term]) for term in text]
             text = [hash_unit.transform(term) for term in text]
-            text = slide_unit.transform(text)
+            text = np.array(text).flatten()
+            text = fix_unit.transform(text.tolist())
```

Comment:
you try to flatten a list of list by converting it into a np array, and then convent it back to list?
Why not directly flatten a list of list using `itertools`?